projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx

```diff
@@ -189,31 +177,7 @@ function SubscriptionsSettings( props ) {
 					/>
 				) }
 			</SettingsCard>
-			<SettingsCard
-				{ ...props }
-				header={ __( 'Messages', 'jetpack' ) }
-				module="subscriptions"
-				saveDisabled={ props.isSavingAnyOption( [ 'subscription_options' ] ) }
-			>
-				<SettingsGroup hasChild disableInOfflineMode module={ subscriptions }>
-					<p className="jp-settings-card__email-settings">
-						{ __(
-							'These settings change the emails sent from your site to your readers.',
-							'jetpack'
-						) }
-					</p>
-					<FormLabel>
-						<span className="jp-form-label-wide email-settings__title">
-							{ __( 'Welcome email message', 'jetpack' ) }
-						</span>
-						<Textarea
-							name={ 'subscription_options' }
-							value={ welcomeMessage }
-							onChange={ changeWelcomeMessageState }
-						/>
-					</FormLabel>
-				</SettingsGroup>
-			</SettingsCard>
+			<SubscriptionSettingsWelcomeMessage { ...props } />
```

Comment:
We should not render it from inside here. Instead do it from `projects/plugins/jetpack/_inc/client/newsletter/index.jsx`.